traffic_stats/build/build_rpm.sh

```diff
@@ -78,6 +78,19 @@ initBuildArea() {
 	go build -v -gcflags "$gcflags" -ldflags "$ldflags" create/create_ts_databases.go || \
 								{ echo "Could not build create_ts_databases binary"; return 1; })
 
+  # compile trafficcontrol-scenes
+  echo "Installing grafana scenes npm dependencies"
+  (cd trafficcontrol-scenes
+    npm i || \
```

Comment:
my build fails on this line with
```
-----  Building traffic_stats ...
/usr/bin/realpath
git is /usr/bin/git
go is /usr/bin/go
/tmp/go/src/github.com/apache/trafficcontrol/build/functions.sh: line 182: type: npm: not found
npm not found in PATH
Error on line 1 of traffic_stats/build/build_rpm.sh
traffic_stats failed: traffic_stats/build/build_rpm.sh
The following subdirectories had errors: 
   traffic_stats
Error on line 1 of ./build/build.sh
```
I rebuilt the builder to be sure. Command I used to test was `./pkg -v -L -S traffic_stats_build`

Reply:
when I build with `./pkg -v -L -S -b traffic_stats_build` it works though... I have no idea why

Reply:
Because you need to rebuild the Docker image of Traffic stats, in the Dockerfile-traffic_stats I added:
`RUN curl -sL https://rpm.nodesource.com/setup_20.x | bash - && \
    yum -y install nodejs`

Reply:
It will install nodejs, so you will get npm for building Grafana Scenes in Traffic stats

Reply:
I think the problem is that Docker won't accept that the current traffic stats builder image is newer than the one it can pull. So it's using the older one. I wonder if you were to rebase this on master if it would still have that problem?